betty/wiki/populator.py

```diff
@@ -244,7 +241,15 @@ async def _image_file_reference(
                     path=image.path,
                     media_type=image.media_type,
                     links=links,
-                    copyright_notice=self._copyright_notice,
+                    copyright_notice=(
+                        await WikimediaCommonsCopyright.new_for_app(
+                            self._ancestry.app,
+                            author=image.image_copyright.author,
+                            url=image.image_copyright.url,
+                        )
+                        if image.image_copyright
+                        else await WikipediaContributors.new_for_app(self._ancestry.app)
```

Comment:
The original issue this PR tries to fix is that we are currently telling people these images are copyright Wikipedia contributors, but that is not the case (the images do not even come from Wikipedia). If we cannot get copyright information from Wikimedia, we should therefore let the image copyright be empty.